src/components/ConfigEditor.test.tsx

```diff
@@ -34,14 +34,15 @@ const setup = async (optionOverrides?: object) => {
       isDefault: false,
       jsonData: {
         host: '',
-        token: '',
         secure: true,
         username: '',
-        password: '',
         selectedAuthType: '',
         metadata: [],
       },
-      secureJsonFields: {},
+      secureJsonFields: {
+        token: false,
+        password: false,
```

Comment:
Just curious, the type for `token` and `password` are both string:
https://github.com/influxdata/grafana-flightsql-datasource/blob/3aa57487a20eda1443288f9ec5899cc2ccfc1e72/src/types.ts#L31-L34

Why do you use `false` instead of `''`?

Reply:
this is `secureJsonFields` not `secureJsonData`

the presence of true or false here dictates whether it was configured as we do not return the token or password to the frontend

Reply:
Ah, got it!